invenio_vocabularies/contrib/affiliations/affiliations.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+#
+# Invenio-Vocabularies is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Vocabulary affiliations."""
+
+from invenio_records_resources.factories.factory import RecordTypeFactory
+
+from ...records.pidprovider import PIDProviderFactory, VocabularyIdProvider
+from ...records.systemfields import BaseVocabularyPIDFieldContext
+from ...services.permissions import PermissionPolicy
+from .config import AffiliationsSearchOptions, service_components
+from .schema import AffiliationSchema
+
+affiliation_record_type = RecordTypeFactory(
```

Comment:
Shorter as it's already in a module and package both named affiliations.
```suggestion
record_type = RecordTypeFactory(
```